local-cluster/src/integration_tests.rs

```diff
@@ -71,6 +71,7 @@ pub const AG_DEBUG_LOG_FILTER: &str = "error,solana_core::replay_stage=info,\
         solana_core::block_creation_loop=trace,\
         solana_votor=trace,\
         solana_votor::vote_history_storage=info,\
+        solana_core::sigverifier::bls_sigverifier=trace,\
```

Comment:
why is this necessary, cannot we set it via ENV variable RUST_LOG?

Reply:
This is mostly for debugging local-cluster tests.